src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -70,6 +70,36 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    Platform manifest needs to include all System*Native items from 3.1 release. This target adds
+    them explicitly, to fill the gap that appeared when the names of these libraries were changed
+    in 5.0 to conform to a specific Android naming convention. The library name change was a fix
+    for https://github.com/dotnet/runtime/issues/33118. For more details of the issue and fix see
+    https://github.com/dotnet/runtime/issues/33450.
+  -->
+  <ItemGroup>
+    <_pastShimFiles Include="System.Globalization.Native.dylib" />
+    <_pastShimFiles Include="System.Globalization.Native.so" />
+    <_pastShimFiles Include="System.IO.Compression.Native.a" />
+    <_pastShimFiles Include="System.IO.Compression.Native.dylib" />
+    <_pastShimFiles Include="System.IO.Compression.Native.so" />
+    <_pastShimFiles Include="System.Native.a" />
+    <_pastShimFiles Include="System.Native.dylib" />
+    <_pastShimFiles Include="System.Native.so" />
+    <_pastShimFiles Include="System.Net.Http.Native.a" />
+    <_pastShimFiles Include="System.Net.Http.Native.dylib" />
+    <_pastShimFiles Include="System.Net.Http.Native.so" />
+    <_pastShimFiles Include="System.Net.Security.Native.a" />
+    <_pastShimFiles Include="System.Net.Security.Native.dylib" />
+    <_pastShimFiles Include="System.Net.Security.Native.so" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.Apple.a" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.Apple.dylib" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.a" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.dylib" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.so" />
+    <SharedFrameworkRuntimeFiles Include="@(_pastShimFiles)" TargetPath="runtimes/" />
```

Comment:
I do like the pattern suggestion from https://github.com/dotnet/runtime/pull/33974#discussion_r396651457. (Applied some of the whitespace I'd normally add to it and pasted below):

```xml
<ItemGroup>
  <_pastShimFilesNames Include="
	System.Native;
	System.Net.Http.Native;
	System.Security.Cryptography.Native.OpenSsl;
	System.Gobalization.Native" />

  <_pastShimFiles Include="@(_pastShimFilesNames->'%(Identity).so')" />
  
  <_pastShimFilesNames Include="System.Security.Cryptography.Native.Apple" />

  <_pastShimFiles Include="@(_pastShimFilesNames->'%(Identity).a')" />
  <_pastShimFiles Include="@(_pastShimFilesNames->'%(Identity).dylib')" />

  <SharedFrameworkRuntimeFiles Include="@(_pastShimFiles)" TargetPath="runtimes/" />
</ItemGroup>
```

Now that it's written and won't be touched again, though, not a big point in changing it.

Reply:
> I do like the pattern suggestion from [#33974 (comment)](https://github.com/dotnet/runtime/pull/33974#discussion_r396651457). 

Yes, that suggestion was liked by everyone 😃 unfortunately, it did miss few cases, and included some files that shouldn't be there. Fixing those created a code that is much less readable. I'd prefer a simple static list, but perhaps a semi-colon delimited one, instead of separate include statements.

Reply:
There's a case to be made for individual include statements... having them self-contained to a particular line of code means there's no difference between begin/end and the middle (`"`, `" />`, vs `;`) to deal with, which avoids some common typos and harder than necessary merge conflicts. (At the cost of worse conflicts when items are simply renamed. And I do think semicolon lists are somewhat easier to read.) Isn't something that probably matters in the long run, but wanted to put that thought out there. 🙂

Reply:
Yeah, I pondered on this a bit before updating the PR. Our codebase is mostly using individual statements. I figured it's easier to merge. Not that we anticipate changes in this particular piece of code, but still it's good to be consistent.